src/_zkapauthorizer/tests/test_storage_protocol.py

```diff
@@ -154,13 +154,13 @@ def setUp(self):
         )
 
         self.clock = Clock()
-        self.anonymous_storage_server = self.useFixture(
+        self.storage = self.useFixture(
             AnonymousStorageServer(self.clock),
-        ).storage_server
+        )
```

Comment:
I don't see any uses of this attribute that don't refer to `.backend`/`.storage_server`, so I'd be inclined to keep the test attribute referring to the `StorageServer` attribute (though, I guess it'd make sense to drop the `anonymous_` prefix here).

Reply:
I think the `anonymous` is a good component since this attribute lives alongside `self.server` which is our ZKAPAuthorizerStorageServer instance.  Maybe `anonymous_server`?

I guess I'm okay with the "server" part since Tahoe is using that word ... but it's less of a server in 1.17.1.  It used to be a thing you could speak Foolscap at (I guess .. sort of .. you needed a Broker and stuff ...) whereas now it's just a "regular" Python object with regular methods.

Reply:
Oh, I guess it isn't clear from the diff included here, but it was named `annonymous_storage_server`. I was proposing naming `storage_server`, given that the type of the object it is storing is `StorageServer`.